lib/stream.js

```diff
@@ -35,6 +35,10 @@ const {
   streamReturningOperators,
   promiseReturningOperators,
 } = require('internal/streams/operators');
+
+const {
+  ERR_ILLEGAL_CONSTRUCTOR
+} = require('internal/errors').codes;
```

Comment:
nit: to be consistent with other modules.
```suggestion
const {
  codes: {
    ERR_ILLEGAL_CONSTRUCTOR,
  },
} = require('internal/errors');
```